src/modules/Elsa.Workflows.Management/Contracts/IWorkflowReferenceQuery.cs

```diff
@@ -0,0 +1,15 @@
+namespace Elsa.Workflows.Management;
+
+/// <summary>
+/// Finds all latest versions of workflow definitions that reference a specific workflow definition.
+/// </summary>
+public interface IWorkflowReferenceQuery
```

Comment:
in the implementation of this  we should prevent loading of workflow definition ids that were already being loaded for 2 reasons 

- performance 
- circular referencing

Reply:
Great catch. Fixed.